traffic_portal/app/src/common/modules/form/cacheGroup/form.cacheGroup.tpl.html

```diff
@@ -85,18 +85,20 @@
             <div class="form-group" ng-class="{'has-error': hasError(cacheGroupForm.latitude), 'has-feedback': hasError(cacheGroupForm.latitude)}">
                 <label class="control-label col-md-2 col-sm-2 col-xs-12">Geo Magnetic Latitude *</label>
                 <div class="col-md-10 col-sm-10 col-xs-12">
-                    <input name="latitude" type="number" class="form-control" ng-model="cacheGroup.latitude" ng-pattern="/^[-+]?[0-9]*\.?[0-9]+$/" required autofocus>
-                    <small class="input-error" ng-show="hasPropertyError(cacheGroupForm.latitude, 'required')">Required Coordinate</small>
-                    <small class="input-error" ng-show="hasPropertyError(cacheGroupForm.latitude, 'pattern')">Invalid Coordinate</small>
+                    <input name="latitude" type="number" class="form-control" ng-model="cacheGroup.latitude" ng-min="-90.0" ng-max="90.0" required autofocus>
+                    <small class="input-error" ng-show="hasPropertyError(cacheGroupForm.latitude, 'required')">Required latitude</small>
+                    <small class="input-error" ng-show="hasPropertyError(cacheGroupForm.latitude, 'min')">Invalid latitude</small>
```

Comment:
how about 

"Invalid latitude. Must be between -90.0 and 90.0"

for the latitude challenged. same for longitude.

Reply:
done